common/changes/@uifabric/experiments/fix-vscode-settings_2018-04-23-23-23.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/experiments",
+      "comment": "Remove wordWrap setting.",
+      "type": "patch"
```

Comment:
Could you change this type to "none"? We don't need to publish a new version for a vscode setting change